### PR TITLE
fix(personaIdentifier lock): Add lockedAt field to handle expired locks on personaIdentifiers (LLC-1877)

### PR DESCRIPTION
**Pull request by ToranSharma**
Closes LLC-1877.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -5,6 +5,9 @@ MODELS_REPO=memory
 STORAGE_REPO=memory
 DEFAULT_TIMEOUT_MS=300000
 
+# Persona Identifier lock expiration time in ms (default 30000)
+IDENTIFIER_LOCK_EXPIRATION_MS=30000
```
Comment (crazy-grizzly) [inline .env.example:9]: @ToranSharma , Nit: So if it has a default value then we can comment out this line? 👀

Comment (ToranSharma) [reply .env.example]: 👍 changed

File: src/repoFactory/options/RefreshIdentifierLockOptions.types.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export interface RefreshIdentifierLockOptions {
```
Comment (lydiamross) [inline src/repoFactory/options/RefreshIdentifierLockOptions.types.ts:1]: @ToranSharma It looks like this isn't used?

**Review by lydiamross — COMMENTED** [review]
Just one comment but otherwise looks good to me 👍

File: src/repoFactory/options/RefreshIdentifierLockOptions.types.ts (hunk 2 above)
Comment (ToranSharma) [reply src/repoFactory/options/RefreshIdentifierLockOptions.types.ts:1]: Good spot, thanks 🙂

**Review by lydiamross — APPROVED** [review]

**Review by lydiamross — APPROVED** [review]
LGTM 👍

**Review by crazy-grizzly — APPROVED** [review]
LGTM 👍

**Pull request merged**